tests/UNITTESTS/README.md

```diff
@@ -52,6 +52,13 @@ target_link_libraries(${TEST_NAME}
         gmock_main
 )
 
+target_compile_definitions(${TEST_NAME}
+     PUBLIC
+        # CMake does not generate configuration options;
+        # if your service has an mbed_lib.json, you must define the configuration parameters here
+        BLE_FEATURE_GATT_SERVER=1
```

Comment:
for unit tests all ble features are alreayd enabled

Reply:
https://github.com/ARMmbed/mbed-os/blob/master/connectivity/FEATURE_BLE/include/ble/common/BLERoles.h#L88

Reply:
I swapped it out for your suggestion: ffcb41c3a483e79e52d09ebe6eb46f19705ddeeb.